index.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
+<html class="no-js" lang="">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <title></title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>CSS Off</title>
+  <link href='https://fonts.googleapis.com/css?family=Oswald:400,300' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Lato:400,100' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Merriweather:400,700,900,300&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Lora:400,700italic,400italic,700' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
+</head>
+<body>
+  <header>
+    <h1><span class="css_spacer">CSS</span> OFF</h1>
+    <p>VOLUME 3-SEPTEMBER 2013</p>
+  </header>
+  <main>
+    <div class="masthead">
+      <article class="barbie">
+        <figure>
+          <img src="img/barbie.png" alt="Barbie rocks">
+        </figure>
+        <div class="barbie_text">
+          <h3>Box Model</h3>
+          <p>The Barbie doll was invented in 1959 by Ruth Handler (co-founder of Mattel), whose own daughter was called Barbara. Barbie was introduced to the world at the American Toy Fair in New York City. Barbies's job was teenage fashion doll. The full name of the first doll was Barbie Millicent Roberts, from Willows, Wisconsin.</p>
+        </div>
+      </article>
+      <article class="position">
+        <figure>
+          <img class="photo" src="img/position_photo.png" alt="Photo of young girl dying of consumption">
+          <img class="shadow" src="img/shadow.png" alt="shadow">
+        </figure>
+        <div class="upshift">
+          <h2>FEATURE</h2>
```

Comment:
Is the string `FEATURE` really more important information then `positioning relatives`? When thinking about semantics, the actual title of the article is more important the a sub-header of the entire page.